viewer/src/utilities/Spinner.ts

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2020 Cognite AS
+ */
+
+// @ts-ignore
+import css from 'raw-loader!./spinnerStyles.css';
```

Comment:
Can this `ts-ignore` be avoided by adding a loader for CSS? How does this line affect bundling - is it embedded, and not a separate file, like e.g. wasm and workers? See https://webpack.js.org/loaders/css-loader/

Reply:
> The css-loader interprets @import and url() like import/require() and will resolve them.

About bundling, with raw loader, it should be just text. Like if everything I have in the file would be written in js string and exported.

![image](https://user-images.githubusercontent.com/4989157/84120564-49b86a80-aa36-11ea-91d1-f5f2fda7718b.png)

But yes, I think I should just add a loader in the webpack config to avoid ts-ignore here. Too easy to break by renaming or something.